src/js/components/Text/stories/TipOnText.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { grommet, Box, Text, Grommet } from 'grommet';
+
+export const TipOnText = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" width="small" pad="small">
```

Comment:
You might want to increase the vertical pad so the tooltip won't overlap with the below example.